frontend/Container.js

```diff
@@ -37,14 +43,42 @@ class Container extends React.Component {
     },
     extraTabs: {[key: string]: () => React$Element},
   };
+  state: State;
+
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      isVertical: (window.innerWidth < 500),
```

Comment:
And just to prove the point it's actually three `500`s :)

Have named it `IS_VERTICAL_BREAKPOINT`